src/components/products/product-details/product-details.tsx

```diff
@@ -0,0 +1,84 @@
+import { useNavigate, useParams } from "react-router-dom";
```

Comment:
Here a nice way of structuring the app is like this:
- /pages folder where the .tsx files that are routes will be stored so ProductList and ProductDetails in this case.
- /components where .tsx files that are used in pages component are stored so ProductListItem.